instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -166,28 +168,45 @@ def _determine_parent_context(
     """
     parent_context = None
 
+    if event_context_extractor:
+        parent_context = event_context_extractor(lambda_event)
+    else:
+        parent_context = _default_event_context_extractor(lambda_event)
+
+    return parent_context
+
+
+def _determine_links(
+    disable_aws_context_propagation: bool = False,
+) -> Optional[Sequence[Link]]:
+    """Determine if a Link should be added to the Span based on the
+    environment variable `_X_AMZN_TRACE_ID`.
+
+
+    See more:
+    https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-lambda.md
+
+    Args:
+        disable_aws_context_propagation: By default, this instrumentation
```

Comment:
Is this mechanism still necessary?

Reply:
Nope. Forgot we decided to remove those.

Reply:
Removed.